src/cljc/matcher_combinators/standalone.cljc

```diff
@@ -3,17 +3,42 @@
             [matcher-combinators.core :as core]
             [matcher-combinators.parser]))
 
+(defn match
+  "Returns a map indicating whether the `actual` value matches the `matcher`.
+
+  `matcher` can be a matcher-combinators matcher, a predicate function of
+  actual, an expression that returns a value, or a literal value.
+
+  `actual` can be an expression that returns the actual value, or a literal.
+
+  Return map includes the following keys:
+
+  - :matcher-combinators.result/type
+    - either :match or :mismatch
+  - :matcher-combinators.result/value
+    - when :matcher-combinators.result/type is :match, binds to the actual value,
+      else a data structure with details about the mismatch."
```

Comment:
this part of the doc needs to be updated

Reply:
Yep - noticed right after I posted that comment :-D

As I was rewriting, I realized that maybe `:mismatch/detail` would be better than `:match/detail`. WDYT? 

I also want to add tests for this before merging. Any objection to me clojure.test-ifying the standalone-test?

Reply:
`:mismatch/detail` does sound better. and :+1: for tests and doing them in clojure.test :)

Reply:
Done.

New question: `:match/detail` currently binds to a `Mismatch` object. I'm thinking maybe that should be converted into a normal data structure. My main concern is tooling outside matcher-combinators dispatching on that type.